_ml-commons-plugin/remote-models/blueprints.md

```diff
@@ -55,19 +55,20 @@ As an ML developer, you can build connector blueprints for other platforms. Usin
+| `description`                                    | String | Yes         | A description of the connector.                                                                                                                                                                                                                                                                                                                                                      |
+| `version`                                        | Integer | Yes         | The version of the connector.                                                                                                                                                                                                                                                                                                                                                        |
+| `protocol`                                       | String | Yes         | The protocol for the connection. For AWS services such as Amazon SageMaker and Amazon Bedrock, use `aws_sigv4`. For all other services, use `http`.                                                                                                                                                                                                                                  |
+| `parameters`                                     | JSON object | Yes         | The default connector parameters, including `endpoint`, `model` and `skip_validating_missing_parameters`. Any parameters indicated in this field can be overridden by parameters specified in a predict request.                                                                                                                                                                     |
+| `credential`                                     | JSON object | Yes         | Defines any credential variables required to connect to your chosen endpoint. ML Commons uses **AES/GCM/NoPadding** symmetric encryption to encrypt your credentials. When the connection to the cluster first starts, OpenSearch creates a random 32-byte encryption key that persists in OpenSearch's system index. Therefore, you do not need to manually set the encryption key. |
+| `actions`                                        | JSON array  | Yes         | Defines what actions can run within the connector. If you're an administrator creating a connection, add the [blueprint]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/blueprints/) for your desired connection.                                                                                                                                                      |
```

Comment:
Either "Defines which actions" or "Defines the actions that can". Is "creating a connection" necessary here?